src/Catalyst.Core.Modules.Dfs.Tests/IntegrationTests/CoreApi/FileSystemApiTest.cs

```diff
@@ -0,0 +1,1013 @@
+            try
+            {
+                var node = (UnixFsNode) ipfs.UnixFsApi.AddFileAsync(path).Result;
+                Assert.Equal("Qmf412jQZiuVUtdgnB36FXFX7xg5V6KEbSJ4dpQuhkLyfD", node.Id);
+                Assert.Equal(0, node.Links.Count());
+                Assert.Equal(Path.GetFileName(path), node.Name);
+            }
+            finally
+            {
+                File.Delete(path);
+            }
+        }
+
+        [Fact]
+        public void AddFile_CidEncoding()
+        {
+            var path = Path.GetTempFileName();
+            File.WriteAllText(path, "hello world");
+            try
+            {
+                var options = new AddFileOptions
+                {
+                    Encoding = "base32"
+                };
+                var node = ipfs.UnixFsApi.AddFileAsync(path, options).Result;
+                Assert.Equal("base32", node.Id.Encoding);
+                Assert.Equal(1, node.Id.Version);
+                Assert.Equal(0, node.Links.Count());
+
+                var text = ipfs.UnixFsApi.ReadAllTextAsync(node.Id).Result;
+                Assert.Equal("hello world", text);
+            }
+            finally
+            {
+                File.Delete(path);
+            }
+        }
+
+        [Fact]
+        public void AddFile_Large()
+        {
+            AddFile(); // warm up
+
+            var path = "star_trails.mp4";
+            var stopWatch = new Stopwatch();
+            stopWatch.Start();
+            var node = ipfs.UnixFsApi.AddFileAsync(path).Result;
+            stopWatch.Stop();
+            
+            // _testOutputHelper.WriteLine("Add file took {0} seconds.", stopWatch.Elapsed.TotalSeconds);
+
+            Assert.Equal("QmeZkAUfUFPq5YWGBan2ZYNd9k59DD1xW62pGJrU3C6JRo", node.Id);
+
+            var k = 8 * 1024;
+            var buffer1 = new byte[k];
+            var buffer2 = new byte[k];
+            stopWatch.Restart();
+            using (var localStream = new FileStream(path, FileMode.Open, FileAccess.Read))
+            {
+                using var ipfsStream = ipfs.UnixFsApi.ReadFileAsync(node.Id).Result;
+                while (true)
+                {
+                    var n1 = localStream.Read(buffer1, 0, k);
+                    var n2 = ipfsStream.Read(buffer2, 0, k);
+                    Assert.Equal(n1, n2);
+                    if (n1 == 0)
+                    {
+                        break;
+                    }
+                    
+                    for (var i = 0; i < n1; ++i)
+                    {
+                        if (buffer1[i] != buffer2[i])
+                        {
+                            throw new XunitException("data not the same");
+                        }
+                    }
+                }
+            }
+
+            stopWatch.Stop();
+            
+            // _testOutputHelper.WriteLine("Readfile file took {0} seconds.", stopWatch.Elapsed.TotalSeconds);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)